packages/zoe/tools/types.js

```diff
@@ -77,7 +77,11 @@
  * brandIn/brandOut pair
  *
  * @property {(brandIn: Brand, brandOut: Brand) => ERef<Notifier<PriceQuote>>}
- * getPriceNotifier
+ * getPriceNotifier be notified of the latest PriceQuotes for a given
+ * brandIn/brandOut pair.  Note that these are raw quotes, and not for a
+ * standardized amountIn or amountOut.  The fact that they are not scaled means
+ * that a PriceAuthority can implement quotes for both fungible and non-fungible
+ * brands.
```

Comment:
I think this is slightly confusing still. For the priceAggregator, we assume that the priceNotifier is giving prices per a standard unit, so saying that this is not standardized seems wrong. Maybe we mean "does not necessarily have to be standardized, but is for the priceAggregator"

Reply:
The `priceAggregator` generates quotes with differing `amountIn`s when somebody asks for a `quoteWanted`.  But I have clarified the comment without needing to name the aggregator.

Reply:
Right, but the priceNotifier acts different than `quoteWanted`, right?